redesign-project/src/_includes/_nav.html

```diff
@@ -0,0 +1,20 @@
+<nav class="wrapper-site-nav">
+    <button id="js-nav-trigger" aria-controls="js-site-nav" aria-expanded="false" class="u-button-reset site-nav__trigger"
+        type="button">
+        <span aria-hidden="true" class="menu-icon">
+            <div class="menu-icon__line"></div>
+            <div class="menu-icon__line"></div>
+            <div class="menu-icon__line"></div>
+        </span>
+
+    </button>
+    <ul id="js-site-nav" class="u-list-reset site-nav__list" aria-hidden="false" data-visible="false">
+        <li class="menu_nav"><a>food</a></li>
+        <li class="menu_nav"><a>reservations</a></li>
+        <li class="menu_nav"><a>events</a></li>
+        <li class="menu_nav"><a>shop</a></li>
+        <li class="menu_nav" id="clickable_subscribe"><a><a href="/subscribe">subscribe</a></a></li>
```

Comment:
You ended up with double `a` tags here:

```suggestion
         <li class="menu_nav" id="clickable_subscribe"><a href="/subscribe">subscribe</a></li>
```